lib/segment/src/data_types/vectors.rs

```diff
@@ -243,6 +264,16 @@ pub struct NamedVector {
     pub vector: VectorType,
 }
 
+/// Vector data with name
+#[derive(Debug, Deserialize, Serialize, JsonSchema, Clone)]
+#[serde(rename_all = "snake_case")]
+pub struct SparseNamedVector {
```

Comment:
```suggestion
pub struct NamedSparseVector {
```

Otherwise it means that the naming is sparse :)

Reply:
+1